src/ucp/rma/basic_rma.c

```diff
@@ -306,7 +306,19 @@ ucs_status_t ucp_get_nbi(ucp_ep_h ep, void *buffer, size_t length,
 
 ucs_status_t ucp_worker_fence(ucp_worker_h worker)
 {
-    return UCS_ERR_UNSUPPORTED;
+    unsigned rsc_index;
+
+    for (rsc_index = 0; rsc_index < worker->context->num_tls; ++rsc_index) {
+        if (worker->ifaces[rsc_index] == NULL) {
+            continue;
+        }
+
+        while (uct_iface_fence(worker->ifaces[rsc_index], 0) != UCS_OK) {
```

Comment:
@xinzhao3 flush test in part of RMA UCp tests. we can add a separate fence test based on ucp atomics, like it you did for UCT. Also, note that currently none of the transports would actually return NO_RESOURCE, so the pending queue flow will not be tested.